tests/integration_tests/test_collection_embedding_function.py

```
@@ -384,6 +390,454 @@ def test_get_or_create_collection_both_none_error(self, db_client):
+            def get_config(self) -> Dict[str, Any]:
+                return {"dimension": self._dimension}
+
+            @staticmethod
+            def build_from_config(config: Dict[str, Any]) -> "UsableEmbeddingFunction":
+                return UsableEmbeddingFunction(dimension=config.get("dimension", 3))
+
+        # Create collection
+        custom_ef = UsableEmbeddingFunction(dimension=3)
+        config = HNSWConfiguration(dimension=3, distance="cosine")
+
+        created_collection = db_client.create_collection(
```

Ruff (0.14.13)

Local variable `created_collection` is assigned to but never used

Remove assignment to unused variable `created_collection`

(F841)